### PR TITLE
fleetctl: Support service uptime as part of a list-units output

### LINKED ISSUE #1128: Feature Request -- Display service uptime as part of a list-units output
We really enjoy using fleetctl to perform cluster related operations, but during debugging sessions we've seen a need to know how long a service has been running in case services are bouncing, it would be nice to see the uptime listed in addition to the service/unit name, Machine, Active/State and Sub

Would this be a possible feature add or is there an alternate way to easily get the same info for service/units running across an entire cluster?

### PR DESCRIPTION
**Pull request by wuqixuan**
We can get the uptime from systemctl, the resolve is like below:
localhost # /home/wood/fleet/fleetctl list-units
UNIT                    MACHINE                         ACTIVE  SUB     UPTIME
world.service           06ecd4f7.../192.168.122.30      active  running 2015-07-06 07:38:38 AM, Since 11m39s
world2.service          1d3430ef.../192.168.122.31      active  running 2015-07-06 07:48:24 AM, Since 1m54s
world_glob.service      06ecd4f7.../192.168.122.30      active  running 2015-07-06 07:48:00 AM, Since 2m18s
world_glob.service      1d3430ef.../192.168.122.31      active  running 2015-07-06 07:47:59 AM, Since 2m18s

Fixed #1128

### CODE REVIEW

File: fleetctl/list_units.go [hunk 1]
```diff
@@ -90,6 +91,14 @@ Or, choose the columns to display:
 			}
 			return us.Hash
 		},
+		"uptime": func(us *schema.UnitState, full bool) string {
+			if us == nil || us.SystemdActiveState != "active" {
+				return "-"
+			}
+			tm := time.Unix(0, int64(us.SystemdActiveEnterTimestamp)*1000)
```
Comment (bcwaldon) [inline fleetctl/list_units.go]: why this instead of `time.Unix(int64(us.SystemdActiveEnterTimestamp), 0)`?

File: fleetctl/list_units.go [hunk 2]
```diff
@@ -90,6 +91,14 @@ Or, choose the columns to display:
...
+			duration := time.Now().Sub(tm)
+			return fmt.Sprintf("%s, Since %ss", tm.Format("2006-01-02 03:04:05 PM"), strings.Split(duration.String(), ".")[0])
```
Comment (bcwaldon) [inline fleetctl/list_units.go]: pull `tm.Format("2006-01-02 03:04:05 PM")` into a global `var` block, and add the timezone

File: systemd/manager.go [hunk 3]
```diff
@@ -238,6 +238,17 @@ func (m *systemdUnitManager) GetUnitStates(filter pkg.Set) (map[string]*unit.Uni
 		states[name] = us
 	}
 
+	// add Active enter time to UnitState
+	for name, us := range states {
+		prop, err := m.systemd.GetUnitProperty(name, "ActiveEnterTimestamp")
```
Comment (bcwaldon) [inline systemd/manager.go]: Is this not available in the properties available in `getUnitState`? making an extra dbus call per unit is pretty expensive...

**Comment by dongsupark** [comment]
Closed in favor of https://github.com/coreos/fleet/pull/1669

**Pull request closed without merging**